package-lock.json

```diff
@@ -1,7 +1,14120 @@
 {
   "name": "@snyk/kubernetes-monitor",
+  "lockfileVersion": 2,
```

Comment:
I think you're running `npm` version 7 which most of us don't use yet, could you ensure we're still running version 6? I think most of our repos are not yet ready for version 7, even though it looks like `snyk-monitor` passed the tests so it will likely be OK!